examples/storm-perf/src/main/java/org/apache/storm/perf/ConstSpoutIdBoltNullBoltTopo.java

```diff
@@ -76,6 +76,8 @@ public static StormTopology getTopology(Map<String, Object> conf) {
     public static void main(String[] args) throws Exception {
         int runTime = -1;
         Config topoConf = new Config();
+        topoConf.put(Config.TOPOLOGY_SPOUT_RECVQ_SKIPS, 8);
```

Comment:
Higher numbers for that setting are only useful for very low latency spouts. The default setting there assumes that you are likely to be using Kafka/Hdfs kind of spouts which have higher latencies (amount of time spent inside nextTuple() ).